### ISSUE TITLE
New HUD style (WIP)

### ISSUE THREAD

**Issue by Gonimy-Vetrom** [Mogli12/AIVehicleExtension#16]
I was a little changed the style of the buttons. If you like, you can use it.
https://drive.google.com/file/d/0B_zrlMUSl3roamJUTzdrM2FrM1k/view?usp=sharing

**Comment by Mogli12** [comment]
Thanks for the new icons. I added them to Version 0.2.0.0

**Comment by Gonimy-Vetrom** [comment]
Little changes  
https://drive.google.com/file/d/0B_zrlMUSl3robXhsUGIySWRBbjA/view?usp=sharing

**Comment by Mogli12** [comment]
Very nice! Thanks!

**Comment by Gonimy-Vetrom** [comment]
![newicons](https://user-images.githubusercontent.com/8200153/28369246-b55ad11a-6c9f-11e7-99c4-98c37337190f.jpg)
Hey. I'm trying to create sharper icons. But I can not find such twists in the scientific literature. Can you send me such a picture?
I've got these pictures ...
![image249](https://user-images.githubusercontent.com/8200153/28369424-37a94322-6ca0-11e7-94a7-1bc1ffe3c417.jpg)
![img-m6drbs](https://user-images.githubusercontent.com/8200153/28369426-37b0f324-6ca0-11e7-8360-3f3f74fbfd61.png)
![image_62](https://user-images.githubusercontent.com/8200153/28369425-37ab9e42-6ca0-11e7-8fce-cec7b6169fea.jpg)

**Comment by Mogli12** [comment]
Hi,
I will try to sketch some images with a single spur and arrows for each turn mode. By the way, the picture with Cyrillic looks interesting.
Thanks and best regards,
Stefan

**Comment by Gonimy-Vetrom** [comment]
Hey. Do you want to change the style of icons?

**Comment by Mogli12** [comment]
Yes, if it makes it easier to understand. Maybe you try out the turn maneuvers ingame.

**Comment by Gonimy-Vetrom** [comment]
![Screenshot_3](https://user-images.githubusercontent.com/8200153/56386992-e369fa80-622b-11e9-98a1-faf236d5bb1c.jpg)
These are icons from the previous version, only without a black background. I can change the color to any. What color will we do? Tell me what icons you need to draw. In order not to do extra work. You can send a list of names from the folder "DDS".

**Comment by Mogli12** [comment]
Hi,

Maybe white color with transparent background fits best to FS19?

Regards,
Stefan

**Comment by Gonimy-Vetrom** [comment]
[dds.zip](https://github.com/Mogli12/AIVehicleExtension/files/3195088/dds.zip)
Hey. Check please. This color is like or not. I did not change the style. Only color.

**Comment by Gonimy-Vetrom** [comment]
![Screenshot_4](https://user-images.githubusercontent.com/8200153/57983255-55ff0d80-7a58-11e9-9163-baf3aad05aa1.jpg)

**Comment by Gonimy-Vetrom** [comment]
[dds.zip](https://github.com/Mogli12/AIVehicleExtension/files/3195499/dds.zip)

**Comment by Gonimy-Vetrom** [comment]
[dds.zip](https://github.com/Mogli12/AIVehicleExtension/files/3219702/dds.zip)
Made a few more icons.

**Comment by Gonimy-Vetrom** [comment]
![Screenshot_4](https://user-images.githubusercontent.com/8200153/58374280-66c1ee80-7f44-11e9-9ba1-bcf39f31960e.jpg)
big_headland and small_headland
Can these icons do this?

**Comment by Mogli12** [comment]
Yes. I used the tool in my icon because the small headland ignores the tool width.

**Comment by Gonimy-Vetrom** [comment]
Now all the icons are ready. But if you have any suggestions, say it and I will fix it.
[FS19_AIVehicleExtensionDDS.zip](https://github.com/Mogli12/AIVehicleExtension/files/3224496/FS19_AIVehicleExtensionDDS.zip)
Temporarily made the icons large (256*256), so that it was clearly visible. But probably you need to do as before (64*64). Does that affect performance?

**Comment by Mogli12** [comment]
Hi,
Many thanks. I added the new icons to version 0.0.2.9. Additionally, I made the background of the HUD less transparent. It looks awesome now.
Regards. Stefan
